test/e2e/fixture/notification/actions.go

```diff
@@ -25,3 +27,12 @@ func (a *Actions) Then() *Consequences {
 	time.Sleep(1 * time.Second)
 	return &Consequences{a.context, a}
 }
+
+func (a *Actions) Healthcheck() *Actions {
+	a.context.t.Helper()
+	_, err := fixture.DoHttpRequest("GET",
+		"/metrics",
```

Comment:
Use `/healthz` or change function name to `IsServingMetrics` to reflect more what the function is doing.

Reply:
Make sense 👍